eo-runtime/pom.xml

```diff
@@ -35,7 +35,7 @@ SOFTWARE.
   <description>EO runtime library</description>
   <properties>
     <!--  This is required for later correct replacement of argline  -->
-    <argLine/>
+    <stack-size>256M</stack-size>
```

Comment:
I think it's too much. I've tried locally and tests become pass with:
```xml
<stack-size>2M</stack-size>
```
As far as I know, the default value is `1M`

Reply:
@volodya-lombrozo yes, it is. Maybe more size(256) can be needed in the future?

Reply:
@volodya-lombrozo well, I think the same. Let it be.